camera/camera_manager.cc

```diff
@@ -62,9 +66,10 @@ std::optional<int> CameraManager::follow_v4l_path(const std::string& path) const
   const std::string v4l_path = fs::canonical(path, ec).string();
   const int index = v4l_path.back() - '0'; // note that this will break for >10 cameras
   if (ec) {
-    return std::nullopt;
-  } else {
     return index;
+  } else {
+    std::cerr << ec << std::endl;
```

Comment:
how descriptive are these error codes? enough to tell that it's coming from the camera manager attempting to follow a v4l path?